worker/include/RTC/Consumer.hpp

```diff
@@ -83,6 +83,12 @@ namespace RTC
 		// RTP counters.
 		RTC::RtpDataCounter transmittedCounter;
 		RTC::RtpDataCounter retransmittedCounter;
+		// RTP sequence number and timestamp.
+		uint16_t seqNum{ 0 };
+		uint32_t rtpTimestamp{ 0 };
+		uint16_t lastRecvSeqNum{ 0 };
+		uint32_t lastRecvRtpTimestamp{ 0 };
+		bool syncRequired{ true };
```

Comment:
Should this be true?

Reply:
We want the very first packet being relayed to have a fresh new sequence number and timestamp, otherwise we cannot calculate it out of a 'non-received' packet.

Reply:
right